crates/ruma-events/tests/ui/08-enum-invalid-path.stderr

```diff
@@ -9,3 +9,15 @@ error[E0433]: failed to resolve: could not find `not` in `ruma_events`
   |
 5 |         "m.not.a.path",
   |         ^^^^^^^^^^^^^^ could not find `not` in `ruma_events`
+
+error[E0639]: cannot create non-exhaustive struct using struct expression
```

Comment:
Do you have an idea what happened here? Looks like the generated code tries to construct a now non-exhaustive struct using a stuct literal.